snapcraft.yaml

```diff
@@ -0,0 +1,32 @@
+name: snapd
+summary: Daemon and tooling that enable snap packages
+description: |
+ Install, configure, refresh and remove snap packages. Snaps are
+ 'universal' packages that work across many different Linux systems,
+ enabling secure distribution of the latest apps and utilities for
+ cloud, servers, desktops and the internet of things.
+
+ This is the CLI for snapd, a background service that takes care of
+ snaps on the system. Start with 'snap list' to see installed snaps.
+version: set-by-version-script-thxbye
```

Comment:
It feels silly that we need to have this while having `version-script`

Reply:
Yeah, snapcraft is silly here.